[DOC-7] Fix links from security/configuration to installation doc (#4034)
EventStore/EventStore: docs/security/configuration.md

> We realise that many users want to try out the latest version with their existing applications, and also run a previous version of EventStoreDB without any security in their internal networks.

Reviewer: ```suggestion
We realize that many users want to try out the latest version with their existing applications, and also run a previous version of EventStoreDB without any security in their internal networks.
```

### Certificate common name

> SSL certificates can be created with a common name (CN), which is an arbitrary string. Usually is contains the DNS name for which the certificate is issued. When cluster nodes connect to each other, they need to ensure that they indeed talk to another node and not something that pretends to be a node. Therefore, EventStoreDB expects the connecting party to have a certificate with a pre-defined CN `eventstoredb-node`.

Reviewer: ```suggestion
SSL certificates can be created with a common name (CN), which is an arbitrary string. Usually it contains the DNS name for which the certificate is issued. When cluster nodes connect to each other, they need to ensure that they indeed talk to another node and not something that pretends to be a node. Therefore, EventStoreDB expects the connecting party to have a certificate with a pre-defined CN `eventstoredb-node`.
```

Usually, you'd need to change rights for each certificate file to prevent the "permissions are too open" error.

You can do it by running the following command:

```bash
chmod 600 [file]
```

:::

#### Generating the CA certificate

> As the first step CA certificate needs to be generated. It'll need to be trusted for each of the nodes and client environment.

Reviewer: ```suggestion
As the first step, a CA certificate needs to be generated. It'll need to be trusted for each of the nodes and client environment.
```

By default, the tool will create the `ca` directory in the `certs` directory you created. Two keys will be generated:

- `ca.crt` - public file that need to be used also for the nodes and client configuration,

> - `ca.key` - private key file that should be used only in the nodes configuration (**Do not copy it to client environment**).

Reviewer: ```suggestion
- `ca.key` - private key file that should be used only in the nodes configuration (**Do not copy it to client environments**).
```

> You can customise generated cert by providing following params:

Reviewer: ```suggestion
You can customize the generated cert by providing the following parameters:
```

You need to generate certificates signed by the CA for each node. They should be installed only on the specific node machine.

By default, the tool will create the `ca` directory in the `certs` directory you created. Two keys will be generated:

- `node.crt` - the public file that needs to be also used for the nodes and client configuration,

> - `node.key` - the private key file that should be used only in the node's configuration (**Do not copy it to client environment**).

Reviewer: ```suggestion
- `node.key` - the private key file that should be used only in the node's configuration (**Do not copy it to client environments**).
```

> You can customise generated cert by providing following params:

Reviewer: ```suggestion
You can customize the generated cert by providing the following parameters:
```

      - ./certs:/certs
```

See more in the [complete sample of docker-compose secured cluster configuration.](../installation.md#use-docker-compose)

## Certificate installation on a client environment

To connect to EventStoreDB, you need to install the auto-generated CA certificate file on the client machine (e.g. machine where the client is hosted, or your dev environment).

#### Linux (Ubuntu, Debian)

> 1. Copy auto-generated CA file to dir `/usr/local/share/ca-certificates/`, e.g. using command:

Reviewer: ```suggestion
1. Copy the auto-generated CA file to `/usr/local/share/ca-certificates/`, e.g. using command:
```

```bash
sudo cp ca.crt /usr/local/share/ca-certificates/event_store_ca.crt
```

2. Update the CA store:

```bash
sudo update-ca-certificates
```

#### Windows

> 1. You can manually import it to the local CA cert store through `Certificates Local Machine Management Console`. To do that select **Run** from the **Start** menu, and then enter `certmgr.msc`. Then import certificate to `Trusted Root Certification`.

Reviewer: ```suggestion
1. You can manually import it to the local CA cert store through `Certificates Local Machine Management Console`. To do that select **Run** from the **Start** menu, and then enter `certmgr.msc`. Then import the certificate to `Trusted Root Certification`.
```
